src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemSettings.scala

```diff
@@ -0,0 +1,50 @@
+package nl.knaw.dans.easy.stage.fileitem
+
+import java.io.File
+
+import nl.knaw.dans.easy.stage.lib.Props._
+import org.joda.time.DateTime
+
+case class FileItemSettings(sdoSetDir: File,
+                            // only used for EasyStageFileItem.run,
+                            // not for createFileSdo as called by EasyStageDataset
```

Comment:
EasyStageFileItem uses the companion object, EasyStageDataset creates the object directly using more of the default values than the companion object.